twofactor_privacyidea/lib/Provider/TwoFactorPrivacyIDEAProvider.php

```diff
@@ -110,6 +110,20 @@ private function getAppValue($key, $default)
         return $this->config->getAppValue('twofactor_privacyidea', $key, $default);
     }
 
+    private function log($level, $message)
+    {
+    	$context = ["app" => "privacyIDEA"];
+    	if($level === 'debug'){
+    		return $this->logger->debug($message, $context);
+	    }
+	    if($level === 'info'){
```

Comment:
The formatting is a bit strage (indentation).
Do we have a mixture of spaces and tabs?